### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by practicalswift**
Fix typos.

### CODE REVIEW

File: src/validation.cpp [hunk 1]
```diff
@@ -2577,7 +2577,7 @@ bool CChainState::ActivateBestChain(CValidationState &state, const CChainParams&
         if (GetMainSignals().CallbacksPending() > 10) {
             // Block until the validation queue drains. This should largely
             // never happen in normal operation, however may happen during
-            // reindex, causing memory blowup  if we run too far ahead.
+            // reindex, causing memory blow up if we run too far ahead.
```
Comment (MeshCollider) [inline src/validation.cpp]: Technically this was already correct and perhaps slightly more natural, I'd remove this change IMO :)

Comment (practicalswift) [reply src/validation.cpp]: Good point! Fixed! :-)

**Comment by promag** [comment]
ACK 1340eda.

**Comment by MeshCollider** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/12283/commits/1340eda3b7b6ca2789d6ec65dad72ee4c3844661

**Comment by fanquake** [comment]
utACK 1340eda, assuming the whitespace changes don't break anything else ready for merge..

**Review by randolf — APPROVED** [review]

**Pull request merged**